packages/framework/src/Resources/views/Admin/Content/Brand/list.html.twig

```diff
@@ -3,34 +3,38 @@
 {% block title %}- {{ 'Brand overview'|trans }}{% endblock %}
 {% block h1 %}{{ 'Brand overview'|trans }}{% endblock %}
 
+{% block btn %}
```

Comment:
This will look pretty weird in online stores with many domains - the button will be too far from the grid imo.

![image](https://user-images.githubusercontent.com/10008612/61797965-c3e4fe00-ae28-11e9-934d-20e6c63f1704.png)

Reply:
Ok this can be reverted.

Reply:
Fixed in 5d44d49ba